src/declarations/constants.ts

```diff
@@ -2,3 +2,21 @@ export const APPLICATION_INSIGHTS_SDK_VERSION = "3.0.0-beta.0";
 export const DEFAULT_BREEZE_ENDPOINT = "https://dc.services.visualstudio.com";
 export const DEFAULT_LIVEMETRICS_ENDPOINT = "https://rt.services.visualstudio.com";
 export const DEFAULT_LIVEMETRICS_HOST = "rt.services.visualstudio.com";
+export const NON_EU_CONNECTION_STRING =
```

Comment:
I'm planning to get rid of declarations folder entirely, please move Statsbeat related types to metrics/statsbeat/types.ts